systemtest/src/main/java/io/strimzi/systemtest/utils/ClientUtils.java

```diff
@@ -236,15 +325,240 @@ public static String generateRandomConsumerGroup() {
+     */
+    public static KafkaClients getInstantPlainClients(TestStorage testStorage, String bootstrapServer) {
+        return getInstantPlainClientBuilder(testStorage, bootstrapServer).build();
+    }
+
+    /**
+     * Retrieves an instance of {@link KafkaClients} for tls communication with Kafka brokers, targeting port 9093 and
+     * leveraging the {@code getInstantTlsClientBuilder} method for initial configuration.
+     *
+     * @param testStorage The {@link TestStorage} instance providing necessary configurations.
+     * @return build {@link KafkaClients}.
+     */
+    public static KafkaClients getInstantTlsClients(TestStorage testStorage) {
+        return getInstantTlsClientBuilder(testStorage, KafkaResources.tlsBootstrapAddress(testStorage.getClusterName())).build();
+    }
+
+    /**
+     * Retrieves an instance of {@link KafkaClients} for tls communication with Kafka brokers,
+     * leveraging the {@code getInstantTlsClientBuilder} method for initial configuration.
+     *
+     * @param testStorage The {@link TestStorage} instance providing necessary configurations.
+     * @param bootstrapServer is the exact address including port (e.g., source-cluster-kafka-bootstrap:9093)
+     * @return build {@link KafkaClients}.
+     */
+    public static KafkaClients getInstantTlsClients(TestStorage testStorage, String bootstrapServer) {
+        return getInstantTlsClientBuilder(testStorage, bootstrapServer).build();
+    }
+
+    //////////////////////////////////
+    // continuous client builders
+    /////////////////////////////////
+
+    /**
+     * Creates a {@link KafkaClientsBuilder} for continuous Kafka clients using plain (non-TLS) communication,
+     * configuring it with properties specific to continuous operation scenarios. This includes setting up
+     * (default 200 messages), a delay between messages (1000 ms) making ideal transition last by default for around 3-4 minutes.
+     * {@link TestStorage#getContinuousProducerName()}  is used for naming producer Job and
+     * {@link TestStorage#getContinuousConsumerName()}  for naming consumer Job. Finally,
+     * {@link TestStorage#getContinuousTopicName()}  is used as Topic target by attempted message transition.
+     *
+     * @param testStorage The {@link TestStorage} instance providing necessary configurations.
+     * @return A configured {@link KafkaClientsBuilder} instance ready for creating Kafka clients for continuous
+     * operations with plain communication, ready for further customization.
+     */
+    public static KafkaClientsBuilder getContinuousPlainClientBuilder(TestStorage testStorage) {
+        return new KafkaClientsBuilder()
+            .withBootstrapAddress(KafkaResources.plainBootstrapAddress(testStorage.getClusterName()))
+            .withNamespaceName(testStorage.getNamespaceName())
+            .withMessageCount(testStorage.getContinuousMessageCount()) // default 200
+            .withDelayMs(1000)
```

Comment:
Should we have a default in test storage?

Reply:
yeah this is used to avoid writing it in each test